src/main/kotlin/nl/info/zac/productaanvraag/ProductaanvraagService.kt

```diff
@@ -420,10 +416,12 @@ class ProductaanvraagService @Inject constructor(
     private fun deleteInboxDocument(documentUUID: UUID) =
         inboxDocumentenService.find(documentUUID).ifPresent { inboxDocumentenService.delete(it.id) }
 
-    private fun findZaaktypeByIdentificatie(zaaktypeIdentificatie: String) =
-        ztcClientService.listZaaktypen(configuratieService.readDefaultCatalogusURI())
-            .firstOrNull { it.identificatie == zaaktypeIdentificatie }
-
+    /**
+     * Handles a productaanvraag-Dimpact [ModelObject] by creating a zaak and starting a CMMN process for it
+     * if there is a zaakafhandelparameters configured for the productaanvraagtype.
+     * Else it will create an 'inbox productaanvraag'.
+     * Note that support for starting a BPMN process from a productaanvraag is not implemented yet.
+     */
```

Comment:
I would check if the below suppressed warnings are still valid after refactoring. It seems that 
- there is no `RuntimeException` being caught
- the nesting is not very deep
- the method is a lot shorter

Reply:
good one! I did remove a few other suppressed warnings the other day. There's probably a command to check for this..?

>there is no RuntimeException being caught

yes, there is! unfortunately. for another day